app/decorators/user_decorator.rb

```diff
@@ -7,6 +7,18 @@ def current_allocation
     super || ENV["DEFAULT_PROJECT_NAME"] || I18n.t('not_allocated')
   end
 
+  def frontend_level
+    return 'N/A' unless model.frontend_level
+
+    "Frontend #{model.frontend_level.humanize}"
```

Comment:
What do you think about not prepending it with "Frontend" or "Backend"? Leaving it just as:

```suggestion
    model.frontend_level.humanize"
```

Reply:
As we talked on Discord, since the table header is fixed on the top, I used this approach to improve the readability